docs/_data/portland-2022-config.yaml

```diff
@@ -224,15 +224,15 @@ flagsoldout: False
 flagspeakersannounced: False
 flagrunofshow: False
 flaghasschedule: False
-flaghasunconf: True
+flaghasunconf: False
```

Comment:
Aren't we doing these?

Reply:
I confess that I don't really know what this flag does. It was false and I flipped it to true and then I checked the website and noticed some links to the sign-up sheet (which doesn't work now) so I flipped it back to false thinking it'll solve it.... I will happily defer to whatever is needed here.

Reply:
Eh, maybe I'm confused on this one. It is indeed listed in things that *do* change.